### DISCUSSION TITLE
Road map for 4.0

### DISCUSSION THREAD

**Discussion by ronvoluted** [open-source-labs/Svelvet#122] [General]
Can you see you fine folks have [begun work on Svelvet 4](https://github.com/oslabs-beta/Svelvet4), congrats! Would you be willing to share a tentative list of things you'll work on?

I was going to hand-roll some modifications around Svelvet (in wrapper logic, in a fork, etc) but it would be great to see what I may not need to write myself.

**Comment by aubertlone** [comment]
Rovoluted, really great to make your acquaintance!

Yes, we are more than happy to share a list of things we're working on! So sorry for the delay, we are currently busy finishing up MVP presentations, and haven't been monitoring the main Svelvet repo on OSLabs for the last couple days.

We would love to meetup and discuss possible changes/improvements. But regardless, the team will reach out by end of day today with a list of the things we're working on. Thank you again!

**Reply by ronvoluted** [reply]
No rush at all!

**Reply by aubertlone** [reply]
Hey, Ron! I wasn't sure about the best way to contact you. So I went to the website linked on your Github profile, and sent an e-mail to the address listed there. Looking forward to speaking further soon!
